src/components/markdown-slides.js

```diff
@@ -3,7 +3,7 @@ import Slide from './slide';
 import Markdown from './markdown';
 
 const transformStringsIntoJSX = function(strings) {
-  return strings.split(/\n---\n/).map((markdown, index) => (
+  return strings.split(/\r?\n---\r?\n/).map((markdown, index) => (
```

Comment:
I'd propose the newlines to be defined as `(\r\n|\r|\n)` which is more correct, since carriage returns `\r` and line feeds can both be used individually. This would cover all styles of line endings.

Do we also want to introduce `---+` to cover the possibility of separators being longer than necessary?

cc @ebrillhart

Reply:
@kitten

AFAIK, no modern system uses `\r` only, but I can update the PR if needed 🙂

Reply:
Phil knows more about the specific carriage return stuff than I do, so I'll defer to him on that particular issue.

With regards to the longer than necessary separators, I do think it would be a good idea to introduce catches - @debel if you'd like to work that into this PR then feel free, otherwise I'll open a separate issue.